.clang-format

```diff
@@ -0,0 +1,117 @@
+---
+Language:        Cpp
+BasedOnStyle:  Google
+AccessModifierOffset: -1
```

Comment:
I meant to delete all of the lines that were already a match with google style, not just uncomment this line.